github-for-jira.sd.yml

```diff
@@ -297,7 +310,6 @@ environmentOverrides:
         WebServerInstanceVolumeSpaceUtilizationAlarmHigh: null
         WebServerMemoryAlarmHigh: null
         WebServerServiceRespawnAlarm: null
-        HighSeverityAlarmWhenTooManyBackend5xxErrors: null
```

Comment:
This is already overridden above for all envs globally, so redundant here

Reply:
And it turned out to be wrong!

I've cab see HighSeverityAlarmWhenTooManyBackend5xxErrors appeared after deploying to ddev. Surprisingly I don't see LP 5xx alerts which were also overridden in global config but not here.

So going to add explicit overrides for all of these alerts

Reply:
I was wrong again! Just realised there was an alert HighSeverityAlarmWhenTooManyBackend5xxErrors which wasn't overridden in the global config